prow/scripts/provision-vm-compass.sh

```diff
@@ -186,7 +186,7 @@ if [[ -z "${VM_FOR_PREFIX_AND_SUFFIX}" ]]; then
     utils::compress_send_to_vm "${ZONE}" "${VM_PREFIX}${SUFFIX}" "/home/prow/go/src/github.com/kyma-incubator/compass" "~/compass"
 
 
-    KYMA_CLI_VERSION="2.1.3"
+    KYMA_CLI_VERSION="2.2.0"
```

Comment:
Why do you use so old version of Kyma CLI? Kyma CLI 2.9.1 was already released.

Reply:
That's the current version we are adopting in Compass. We have regular tasks to adopt new Kyma versions but it's not a trivial process.